web/src/app/shared/util/taskana.date.ts

```diff
@@ -13,7 +13,7 @@ export class TaskanaDate {
     const dateFormat = 'yyyy-MM-dd';
     const dateLocale = 'en-US';
     const datePipe = new DatePipe(dateLocale);
-    return datePipe.transform(date, dateFormat);
+    return datePipe.transform(date, dateFormat, Intl.DateTimeFormat().resolvedOptions().timeZone);
```

Comment:
This is no needed. const datePipe = new DatePipe(dateLocale); already applies timezone

Reply:
Rollback this line